code/mergeblock/merge.py

```diff
@@ -0,0 +1,86 @@
+#Jakub Podolak for Plastic Monkeys Cansat 2019
+
+import subprocess
+import configparser
+import msgpack
+import time
+import sys
+import re
+
+#function to read paths to sensors' logs from config file
+def get_directories(config):
+    directories = []
+    paths = config.items("PATHS")
+    for key, path in paths:
+        directories.append(path)
+    return directories
+
+#function reads sensors' names from config file
+def get_keys(config):
+    keys = []
+    paths = config.items("PATHS")
+    for key, path in paths:
+        keys.append(key)
+    return keys
+
+#function takes paths to logs as an argument, returns list of the latest readings(int) from all paths
+def get_tails(directories,length):
+    tails = []
+    for x in directories:
+        with open(x, 'rb') as fh:
+            fh.seek(-1*length, 2)
```

Comment:
does it work like that? As far as I remember, it was required to read at least one like before `seek`ing to the end

Reply:
As i can see [here](https://python-reference.readthedocs.io/en/latest/docs/file/seek.html) it is not required, also it works like a charm